Sources/Widgets/Widgets3D/LineWidget/behavior.js

```diff
@@ -0,0 +1,261 @@
+import macro from 'vtk.js/Sources/macro';
+
+const MAX_POINTS = 2;
+
+export default function widgetBehavior(publicAPI, model) {
+  model.classHierarchy.push('vtkLineWidgetProp');
+  let isDragging = null;
+  const direction = [0, 0, 0];
+  const coneBehavior = {
```

Comment:
Can you export it in an other file ? You can take a look at the Constants.js file in WidgetManager folder to have an example.